front/README.md

```diff
@@ -40,26 +40,27 @@ Launches end to end tests.
 It requires:
 
 - Install playwright dependencies `cd ./front/ && yarn playwright install --with-deps`
-- Backend containers to be up: `docker compose up --no-build --detach redis postgres gateway core editoast`
+- Backend containers to be up:
+  `docker compose up --no-build --detach redis postgres gateway core editoast`
 - Running front with `docker compose up --build --detach front`
 
 Now you can run the test with `cd front/ && yarn e2e-tests`.
 
-> [!CAUTION]
-> If you try to run `yarn start` instead of running it through docker, you'll notice it doesn't
-> work because the gateway can't access your local port from inside a container. 2 solutions:
+> [!CAUTION] If you try to run `yarn start` instead of running it through docker, you'll notice it
+> doesn't work because the gateway can't access your local port from inside a container. 2
+> solutions:
```

Comment:
This is broken (you can check the GitHub preview). The line feed is required.
We should probably add this file to the `.prettierignore`.

Reply:
Eh :/ 

Oh well, I just added a rule to ignore `*.md` for now.

(An alternative would be to add two spaces at the end of the line probably, but we'd risk getting it removed by mistake by some editors stripping whitespace at end-of-lines automatically when saving a file.)